src/main/java/com/accesso/challengeladder/controller/MatchController.java

```diff
@@ -35,6 +32,11 @@ public MatchController() throws SQLException, IOException
             return JsonUtil.toJson(matchService.getMatch(req.params(":id")));
         });
 
+        // get information of recent matches
+        get("/recent_matches/:limit", (req, res) -> {
```

Comment:
the resource should be matches and not recent_matches. IMO frontend should paginate the existing response from GET /matches